### ISSUE TITLE
Passing 'true' to the ip validator does not work.

### ISSUE THREAD

**Issue by brettlaforge** [balderdashy/anchor#72]
I've been using waterline as an ORM and I've ran into a issue.

When I pass 'true' to the ip validation rule, anchor attempts to call validator's isIP method with the data's value as the first argument and the rule's value as the second argument.
This works for most validators but not for ip because isIP wants the second parameter to be either the number 4 or the number 6 to use IPv4 or IPv6 validation.

validator.isIP('127.0.0.1', true) -> false
validator.isIP('127.0.0.1', 4) -> true
validator.isIP('127.0.0.1') -> true

In fact I think we are just lucky that the email validator works as well as it does considering it too passes true as the second options argument. I also argue that no secondary argument should be provided as the ipv4 and ipv6 validation exist if you need those.

Maybe a list of rules that need the rule's value as the second argument are necessary?

Example:
var System = waterline.Collection.extend({
        attributes: {
            ip: {
                type: 'string',
                required: true,
                ip: true
            }
        }
    });
Results in a call made to:
validator.isIP('127.0.0.1', true) -> false

Workaround add undefined as the value in your rule.
var System = waterline.Collection.extend({
        attributes: {
            ip: {
                type: 'string',
                required: true,
                ip: undefined
            }
        }
    });
Results in a call made to:
validator.isIP('127.0.0.1', undefined) -> true

**Comment by sailsbot** [comment]
Thanks for posting, @brettlaforge.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
